lms/djangoapps/teams/static/teams/templates/team-profile.underscore

```diff
@@ -1,10 +1,71 @@
 <div class="team-profile">
-    <div class="discussion-module" data-course-id="<%= courseID %>" data-discussion-id="<%= discussionTopicID %>"
-            data-read-only="<%= readOnly %>"
-            data-user-create-comment="<%= !readOnly %>"
-            data-user-create-subcomment="<%= !readOnly %>">
-        <% if ( !readOnly) { %>
-            <a href="#" class="new-post-btn" role="button"><span class="icon fa fa-edit new-post-icon"></span><%= gettext("New Post") %></a>
+    <div class="page-content-main">
+        <div class="discussion-module" data-course-id="<%= courseID %>" data-discussion-id="<%= discussionTopicID %>"
+                data-read-only="<%= readOnly %>"
+                data-user-create-comment="<%= !readOnly %>"
+                data-user-create-subcomment="<%= !readOnly %>">
+            <% if ( !readOnly) { %>
+                <a href="#" class="new-post-btn" role="button"><span class="icon fa fa-edit new-post-icon"></span><%= gettext("New Post") %></a>
```

Comment:
@cptvitamin @muzaffaryousaf has just moved this inside a div. I have logged this issue in https://openedx.atlassian.net/browse/TNL-1930.